modules/json/src/main/scala/gem/json.scala

```diff
@@ -43,6 +45,13 @@ package object json {
       moveLeft(b, 3)
   }
 
+  private implicit class PrismOps[A: Encoder: Decoder, B](p: Prism[A, B]) {
+    def toCodec: (Encoder[B], Decoder[B]) = (
+      Encoder[A].contramap(p.reverseGet),
+      Decoder[A].map(p.unsafeGet(_))
+    )
+  }
+
```

Comment:
When we finally commit to a strategy re: Json we can organize this stuff properly.